roles/matrix-synapse/defaults/main.yml

```diff
@@ -209,7 +209,8 @@ matrix_mautrix_telegram_api_hash: YOUR_TELEGRAM_API_HASH
 # Mautrix telegram public endpoint to log in to telegram
 # Use an uuid so it's not easily discoverable
 matrix_mautrix_telegram_public_endpoint: "/{{ matrix_synapse_macaroon_secret_key | password_hash('sha512', 'telegram') | to_uuid }}"
-
+# Set this to a port number to expose on the host when not using the nginx proxy
+matrix_mautrix_telegram_container_expose_port: false
```

Comment:
If this is meant to be a port number (integer), having a default value which is a boolean is a bit strange.

We'd better use a default value of null (`~` in YAML). This way, it's either an integer or null.

In any case, both what you have now and what I'm proposing as an alternative lead to the same - `matrix_mautrix_telegram_container_expose_port` being different than all other `_expose_port` variables we have (those are true/false values only and don't specify a port). However, it's probably a good idea to break our old consistency and move toward this new model ("port number or null").

However, I'd probaly rename the variable to something that has less of a "this is a boolean" feel.
Perhaps instead of `matrix_mautrix_telegram_container_expose_port`, we could have it be named `matrix_mautrix_telegram_container_exposed_port_number`.

Once this bridge leads the way, we can redo our other `_expose_port` instances to follow suit.